docs/topics/native/native-binary-licenses.md

```diff
@@ -0,0 +1,73 @@
+[//]: # (title: License files for the Kotlin/Native binaries)
+
+Like many other open-source projects, Kotlin relies on third-party work. This means that the Kotlin project also
+includes the code developed not by JetBrains s.r.o. or the Kotlin programming language contributors. Sometimes it is not
+the exact code but derived work, like the code rewritten from C++ to Kotlin.
+
+>  You can find licenses for the third-party work used in Kotlin in our GitHub repository:
+>
+> * [Kotlin compiler](https://github.com/JetBrains/kotlin/tree/master/license/third_party)
+> * [Kotlin/Native](https://github.com/JetBrains/kotlin/tree/master/kotlin-native/licenses/third_party)
+>
+{type="note"}
+
+In particular, the Kotlin/Native compiler produces binaries that can include third-party code, data, or derived work.
+This means that the Kotlin/Native-compiled binaries are subject to the terms and conditions of the corresponding
+licenses.
+
+In practice, if you distribute a Kotlin/Native-compiled [final binary](multiplatform-build-native-binaries.md),
+you should always include necessary license files into your binary distribution. The files in a readable form should be
```

Comment:
The files should be accessible to users of your distribution in a readable form.

Sounds better for me.